visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.component.html

```diff
@@ -16,8 +16,8 @@ <h3 class="header-text">Flattened</h3>
 		</md-list-item>
 
 		<md-list-item
-			ng-repeat="item in $ctrl._viewModel.flatten | orderBy: ['path'] track by $ctrl._viewModel.flatten.indexOf(item)"
-			title="{{::item.path}}"
+			ng-repeat="item in $ctrl._viewModel.flatten | orderBy: ['path']"
+			title="{{::item.path}} track by $ctrl._viewModel.exclude.indexOf(item.path)"
 		>
```

Comment:
```suggestion
			ng-repeat="item in $ctrl._viewModel.flatten | orderBy: ['path']"
			title="{{::item.path}} track by $ctrl._viewModel.flatten.indexOf(item.path)"
		>
```

Reply:
I will include the track by without the title.. Thanx